frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -55,7 +55,8 @@ export interface MultiplexResult {
   disease: {
     name: string;
   };
-  testResult: TestResult;
+  testResult?: TestResult;
```

Comment:
⚠️ had to add this because the pxp endpoint returns
```
{ disease: {
    name: string
  },
 result: string
}
```
instead of the graphQL that returns
```
{ disease: {
    name: string
  },
 testResult: string
}
```
Is there a way to update this in the backend so they are the same?

Reply:
created #3958 for the backend change and #3959 for the frontend changes